tee-worker/ts-tests/integration-tests/di_vc.test.ts

```diff
@@ -88,4 +67,46 @@ describe('Test Vc (direct invocation)', function () {
             await assertVc(context, aliceSubject, res.value);
         });
     });
+    extraAssertions.forEach(({ description, assertion }) => {
+        step(`request vc ${Object.keys(assertion)[0]} (alice)`, async function () {
+            let currentNonce = (await getSidechainNonce(context, teeShieldingKey, aliceSubject)).toNumber();
+            const getNextNonce = () => currentNonce++;
+            const nonce = getNextNonce();
+            const requestIdentifier = `0x${randomBytes(32).toString('hex')}`;
+            console.log(`request vc ${Object.keys(assertion)[0]} for Alice ... Assertion description: ${description}`);
+            const eventsPromise = subscribeToEventsWithExtHash(requestIdentifier, context);
+
+            const requestVcCall = await createSignedTrustedCallRequestVc(
+                context.api,
+                context.mrEnclave,
+                context.api.createType('Index', nonce),
+                new PolkadotSigner(context.substrateWallet.alice),
+                aliceSubject,
+                context.api.createType('Assertion', assertion).toHex(),
+                context.api.createType('Option<RequestAesKey>', aesKey).toHex(),
+                requestIdentifier
+            );
+
+            const res = await sendRequestFromTrustedCall(context, teeShieldingKey, requestVcCall);
+            assert.isTrue(res.do_watch.isFalse);
+            assert.isTrue(res.status.asTrustedOperationStatus[0].isInvalid);
```

Comment:
This will definitely fail here, as indicated in the [comments](https://github.com/litentry/litentry-parachain/pull/2275/files/bc17c91cb992f1877fa4e8b5d323c8e6b4c52320#diff-803593e3a8b9e787b7d0c1864c9da5b19e7bc805f93a9ebd8615cc5bf4823641R150). Therefore, the purpose here is not to test for failure, but to test if DI can properly retrieve the response and trigger an event

Reply:
> OK, I _think_ I get it now -- in that case, please rename the variable to `unconfiguredAssertions` (to make it explicitly that they should be configured but are currently not); then add a [pending test](https://mochajs.org/#pending-tests) and a Linear issue to tackle this at a later point 🙏

Thanks! https://linear.app/litentry/issue/P-328/follow-up-make-unconfiguredassertions-pass-the-test